src/Core/AdminConsole/OrganizationAuth/UpdateOrganizationAuthRequestCommand.cs

```diff
@@ -51,5 +76,64 @@ public async Task UpdateAsync(Guid requestId, Guid userId, bool requestApproved,
                 updatedAuthRequest.RequestIpAddress, deviceTypeAndIdentifier);
         }
     }
-}
 
+    public async Task UpdateAsync(Guid organizationId, IEnumerable<OrganizationAuthRequestUpdate> authRequestUpdates)
+    {
+        await new BatchAuthRequestUpdateProcessor<OrganizationAdminAuthRequest>(
+            await FetchManyOrganizationAuthRequestsFromTheDatabase(organizationId, authRequestUpdates.Select(aru => aru.Id)),
```

Comment:
The only argument here that I think applies is the first one since it makes another call, and I've split that out as suggested here: https://github.com/bitwarden/server/pull/4064/commits/316607f5185633fa7e37a1b7fccfa11f5aa3e868

I left the other two alone, the updates object is direct from the function parameters and the configuration object does not have a lot of logic in it. I think it'd be fine to debug those in the processor constructor if needed.